Modules/Tokens/src/Tokens.Application/Infrastructure/Persistence/Repository/ITokensRepository.cs

```diff
@@ -9,8 +9,8 @@ namespace Backbone.Modules.Tokens.Application.Infrastructure.Persistence.Reposit
 public interface ITokensRepository
 {
     Task Add(Token token);
-    Task<Token> Find(TokenId tokenId);
-    Task<DbPaginationResult<Token>> FindAllWithIds(IEnumerable<TokenId> ids, PaginationFilter paginationFilter, CancellationToken cancellationToken);
+    Task<Token> Find(TokenId tokenId, IdentityAddress? address);
```

Comment:
We should make clear whose address we expect.
```suggestion
    Task<Token> Find(TokenId tokenId, IdentityAddress? activeIdentity);
```

Reply:
changes made